tests/sanity/ignore-2.11.txt

```diff
@@ -23,12 +23,7 @@ plugins/modules/azure_rm_automationaccount_info.py validate-modules:parameter-li
 plugins/modules/azure_rm_availabilityset.py validate-modules:parameter-type-not-in-doc
 plugins/modules/azure_rm_availabilityset_info.py validate-modules:parameter-type-not-in-doc
 plugins/modules/azure_rm_availabilityset_info.py validate-modules:parameter-list-no-elements
-plugins/modules/azure_rm_containerinstance.py validate-modules:doc-type-does-not-match-spec
```

Comment:
@aparna-patil The test/sanity/ignore. 2.12. txt need the same changes. Thank you very much!